.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": ["fbjs", "prettier"],
```

Comment:
Are we adding more of them later?

Reply:
I think not so soon. [fbjs](https://github.com/facebook/fbjs/blob/master/packages/eslint-config-fbjs/index.js) config is quite ok for now. We can always overrides their rules later on